### PR TITLE
RSDK-6031 Remove GetLatestMapInfo from SLAM

### PR DESCRIPTION
**Pull request by jeremyrhyde**
On hold until release.

This PR removes the GetLatestMapInfo endpoint from the SLAM API.

**JIRA Ticket:** https://viam.atlassian.net/browse/RSDK-6031

### CODE REVIEW

**Review by randhid — APPROVED** [review]
Breaking change, so you're going to need Simone to approve i think

File: proto/viam/service/slam/v1/slam.proto [hunk 1]
```diff
@@ -37,14 +37,6 @@ service SLAMService {
     };
   }
 
-  // GetLatestMapInfo returns a message indicating details regarding the
```
Comment (randhid) [inline proto/viam/service/slam/v1/slam.proto:40]: can't comment - line 8 is now unused

Comment (jeremyrhyde) [reply proto/viam/service/slam/v1/slam.proto:40]: Good catch!

**Review by kim-mishra — APPROVED** [review]

**Review by Simberific — APPROVED** [review]
LGTM

**Pull request merged**